client/src/App.js

```diff
@@ -12,7 +12,8 @@ export default class App extends Component {
     };
   }
 
-  addToSavedList = movie => {
+  addToSavedList = (movie) => {
+    console.log(this.state.savedList)
```

Comment:
See if you can fix the functionality so that the same movie is not saved multiple times.